incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -29,6 +31,11 @@ type ConfigReconciler struct {
 	// Forest is the in-memory data structure that is shared with all other reconcilers.
 	Forest *forest.Forest
 
+	// Affected is a channel of event.GenericEvent (see "Watching Channels" in
```

Comment:
Since we only use this for initialization, maybe call this something more limited? Eg, `StartupQueue` or `Kickstarter` or `Igniter` or something like that :)

Reply:
Sure! Changed to Igniter.